pyproject.toml

```diff
@@ -90,19 +90,19 @@ develop = [
     "boto3==1.18.46",
     # tests
     "ujson",
-    "pytest==6.2.5",
-    "pytest-benchmark==3.2.2",
-    "pytest-asyncio==0.18.1",
-    "tox==3.14.0",
-    "sphinx==4.2.0",
+    "pytest==7.1.2",
+    "pytest-benchmark==3.4.1",
```

Comment:
`make benchmark` continues to work

Reply:
However it notices a bug in benchmarks/driver/runner_test.py where we call bulk_index but this is an async function and it's never awaited so the benchmark is bogus. pytest-benchmark does not support asyncio though: https://github.com/ionelmc/pytest-benchmark/issues/66